src/services/entry-service.ts

```diff
@@ -49,8 +88,19 @@ class EntryService {
   }
 }
 
-const processResponse = (response: EntryListResponse) => {
-  return Promise.resolve(response.items.length ? response.items.map(entryData => entryData.sources) : []);
+const getDurationHumanizer = () => {
+  try {
+    const supportedLanguages = humanizeDuration.getSupportedLanguages();
+    const language = navigator.language;
```

Comment:
see [here](https://github.com/kaltura/playkit-js/blob/master/src/utils/locale.js#L13) for cross-browser